src/amo/components/AddonReviewList.js

```diff
@@ -107,6 +107,13 @@ export class AddonReviewListBase extends React.Component {
     return `${this.addonURL()}reviews/`;
   }
 
+  formatReviewBody(text) {
+    const formattedReview = text.split('\n').map((i) => {
+      return <div key={i}>{i}</div>;
```

Comment:
Because these reviews are already inside a `<p>` tag, I'd prefer not putting divs inside them. This is a bit of a hack anyway because the divs are block-level elements.

A better approach would be to convert the newline characters to `<br>` tags. We already have a utility function `nl2br` in `core/utils` you can use for that.